### ISSUE TITLE
"quote" substitutions not working in the preview

### ISSUE THREAD

**Issue by abelsromero** [asciidoctor/asciidoctor-intellij-plugin#193]
**AsciiDoc Plugin Version** 0.19.1

**IDE Name and Version**
IntelliJ IDEA 2017.2.5
Build #IC-172.4343.14, built on September 26, 2017
JRE: 1.8.0_152-release-915-b12 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 7 6.1

The following piece should show the "bold" part in bold in the preview, but it doesn't.
```
[source,java,subs="verbatim,quotes"]
----
System.out.println("Hello *bold* text").
----
```

![image](https://user-images.githubusercontent.com/5781153/31550190-cf61ec54-b030-11e7-8afd-fa24f51af28c.png)

It does not show the `*`, so it's processing the subs, but something odd happens.

Rendering using Asciidoctorj 1.5.6 with the maven-plugin works fine.
![image](https://user-images.githubusercontent.com/5781153/31550172-bf9739dc-b030-11e7-9d70-10263d43ebf2.png)

It also happens with listing blocks (`....`).

**Comment by mojavelinux** [comment]
This just looks like a stylesheet issue. Either a rule is missing or it's being overridden.

**Comment by ahus1** [comment]
There was a problem with improperly installed fonts (probably a bold font missing)? The upcoming 0.20.0 version includes fonts and a proper font-family mapping, and the example from above works.

**Comment by ahus1** [comment]
The 0.20.0 is now available as a preview from the Github releases page of this project: https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases

**Comment by abelsromero** [comment]
I tested the preview version and I am sorry to say that the issue persists. Also the font of the whole text  changes.

**v 0.19.2**
![image](https://user-images.githubusercontent.com/5781153/36065423-d97299dc-0e9a-11e8-9208-79a1c0210c30.png)

**preview v 0.20**
![beta](https://user-images.githubusercontent.com/5781153/36065429-f4f01b1c-0e9a-11e8-81e0-8a3feb8825b9.PNG)

**Comment by ahus1** [comment]
The font change for the whole text is indentional - it now uses and bundles the default asciidoctor fonts. 

I re-created the issue; it only works in development mode, but not when using the packaged plugin. I'll have to investigate.

**Comment by ahus1** [comment]
I tracked this down first to a different handling in Oracle JDK (working) and OpenJDK / JetBrains Runtime (not working). 

Then I found if I remove the default fonts like "Droid Sans Mono" (that are not available locally anyway as they are Google fonts), also OpenJDK works as expected.

Version 0.26.20 includes the fix and is available from the [GitHub releases as a preview](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases).
